api/src/Entity/ContentNode/SingleText.php

```diff
@@ -58,14 +58,25 @@ class SingleText extends ContentNode {
         ],
     ];
 
+    public const DATA_DEFAULT = '{"html":""}';
+
+    public function __construct() {
+        parent::__construct();
+        $this->data = json_decode(self::DATA_DEFAULT, true);
+    }
+
     /**
      * Holds the actual data of the content node
      * (overridden from abstract class in order to add specific validation).
      */
-    #[ApiProperty(example: ['html' => 'my example text'])]
+    #[ApiProperty(
+        default: self::DATA_DEFAULT,
```

Comment:
Does default has any effect at all? I believe to remember that we noticed earlier, that the default property has no effect and hence removed it from our code.

Reply:
It now has an effect.
Without the default here, it uses the value set on the field.
(it seems to detect what is assigned to the field on initialization).
This happens in ContentNode, and is an unusable default for most of the child classes.